strider/fetcher.py

```diff
@@ -150,18 +150,18 @@ async def generate_plan(self):
 
         self.logger.debug({"plan": self.plan})
 
-        # add first partial result
+        # add partial results for each curie that we are given
         for qnode_id, qnode in self.qgraph["nodes"].items():
-            if "id" not in qnode or qnode["id"] is None:
+            if not qnode.get("id", False):
                 continue
-            result = {
-                "node_bindings": {
-                    qnode_id: [{"id": qnode["id"]}]
-                },
-                "edge_bindings": {},
-            }
-            await self.put(result)
-            break
+            for curie in qnode["id"]:
```

Comment:
Do we have a test for this?

Reply:
What do you mean exactly? This line has coverage from a couple of the tests in tests/server/test_server.py. Is there a specific condition you had in mind to test?

Reply:
I mean a test for a qnode with multiple `id`s. Probably it's tied in with the multiple-plan issues.

Reply:
If I make one, it will be failing. Should I still add it to this commit, or wait for your multiple plan issues?

Reply:
That's okay, we'll add a test later.